### ISSUE TITLE
Rotate Gesture

### ISSUE THREAD

**Issue by Limonlol** [icerockdev/moko-maps#4]
Need func or property to disable rotate gesture

**Comment by Alex009** [comment]
in 0.2.0 will be available settings:
```kotlin
data class UiSettings(
    val compassEnabled: Boolean = false,
    val myLocationEnabled: Boolean = false,
    val myLocationButtonEnabled: Boolean = false,
    val indoorLevelPickerEnabled: Boolean = false,
    val scrollGesturesEnabled: Boolean = true,
    val zoomGesturesEnabled: Boolean = true,
    val tiltGesturesEnabled: Boolean = true,
    val rotateGesturesEnabled: Boolean = true,
    val scrollGesturesDuringRotateOrZoomEnabled: Boolean = true
)
```

**Comment by Alex009** [comment]
released in 0.2.0
